Dockerfile.repo

```diff
@@ -9,19 +9,18 @@ RUN echo "$JAVA_HOME/jre/lib/$(arch)/server/" > /etc/ld.so.conf.d/rApache_rJava.
 
 RUN \
   echo "fastestmirror=1" >> /etc/dnf/dnf.conf && \
-  sed -i 's/vault.centos.org/archive.kernel.org/g' /etc/yum.repos.d/* && \
   dnf update -y && \
   dnf upgrade -y && \
   dnf install -y epel-release && \
-  dnf config-manager --set-enabled powertools && \
-  dnf module enable -y postgresql:13 && \
-  dnf install postgresql-server-devel postgresql-devel postgresql -y && \
-  dnf install -y rpm-build which make wget tar httpd-devel libapreq2-devel libcurl-devel protobuf-devel openssl-devel libpng-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel curl-devel libxml2-devel llvm5.0-devel llvm-toolset sudo cmake initscripts && \
```

Comment:
Updated to postgres:15 since 13 wasn't available. I couldn't find protobuf-devel and everything built and worked without it. I couldn't find why we actually needed it.